app/views/submissions/submit_comment.erb

```diff
@@ -1,17 +1,16 @@
 <% if current_user.guest? %>
-  <p> You're not logged in right now.Please login via <a href="/login">GitHub </a> to comment </p>
+  <p> You're not logged in right now. Please login via <a href="/login">GitHub </a> to comment </p>
```

Comment:
It would be good to add a return_path parameter, see  
[app/views/site/how_it_works.erb#L27](https://github.com/exercism/exercism.io/blob/master/app/views/site/how_it_works.erb#L27)